tests/rendering/protos/depth_2/NestedOverwrittenExposedTexture.proto

```diff
@@ -0,0 +1,17 @@
+#VRML_SIM R2022b utf8
+
+EXTERNPROTO "./depth_3/ExposedTexture.proto"
```

Comment:
We should also test without the `./` prefix which is a bit superfluous.
```suggestion
EXTERNPROTO "depth_3/ExposedTexture.proto"
```

Reply:
I've added it precisely because it is superfluous, since the relative path needs to be combined with the location path, it's to ensure it does the correct job of concatenating even if people happen to add it.

>We should also test without the `./`

So should I duplicate all tests and check both versions? Or just mix up the existing tests (so not all are `./`)

Reply:
I believe that simply mixing-up the existing tests is enough.